packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockListMenuContent.tsx

```diff
@@ -9,38 +9,35 @@ import {
   Typography,
 } from "@mui/material";
 import { PopupState } from "material-ui-popup-state/core";
-import { useRef, useState, FunctionComponent } from "react";
+import { useRef, useState, FunctionComponent, useEffect } from "react";
 import { TextField, FontAwesomeIcon } from "@hashintel/hash-design-system";
-import { BlockSuggesterProps } from "../createSuggester/BlockSuggester";
 import { useFilteredBlocks } from "../createSuggester/useFilteredBlocks";
 import { MenuItem } from "../../../shared/ui";
+import { useBlockView } from "../BlockViewContext";
 
 type BlockListMenuContentProps = {
   popupState?: PopupState;
-  blockSuggesterProps: BlockSuggesterProps;
   compatibleBlocks: HashBlock[];
 };
 
 export const BlockListMenuContent: FunctionComponent<
   BlockListMenuContentProps
-> = ({ blockSuggesterProps, popupState, compatibleBlocks }) => {
+> = ({ compatibleBlocks, popupState }) => {
   const [searchQuery, setSearchQuery] = useState("");
   const searchInputRef = useRef<HTMLInputElement>(null);
   const blocks = useFilteredBlocks(searchQuery, compatibleBlocks);
+  const blockView = useBlockView();
 
-  // The essence of this is to autoFocus the input when
-  // the blocklist menu comes up. We have a listener for
-  // character "/". Once that is clicked the MenuItem that has this submenu
-  // is focused and as a result the submenu becomes visible.
-  // Currently this flow introduces a bug where it is difficult to switch to certain blocks
-  // e.g Embed;
-  // @see https://github.com/hashintel/hash/pull/480#discussion_r849594184
-  // Commenting this out till a fix is made
-  // useEffect(() => {
-  // if (popupState?.isOpen) {
-  //   searchInputRef.current?.focus();
-  // }
-  // }, [popupState]);
+  const popupWasOpen = useRef<boolean>(false);
```

Comment:
The problem was this effect overfiring when scrolling the popup window, because the state has a bunch of stuff in it and changes quite often. Updated to only do something when the `isOpen` state has actually changed